tests/test_ingest.py

```diff
@@ -81,18 +87,21 @@ def tearDown(self) -> None:
 class TestRun(TestIngest):
 
     def test_ss2_ingest_to_upload(self):
-        runner = self.ingest_and_upload_only('SS2')
+        self.ingest_and_upload_only('SS2')
```

Comment:
Could the `SS2` name be made a constant?
We might add additional datasets to the test suite.

Reply:
Done